ballerina/variable_validator.bal

```diff
@@ -130,9 +142,15 @@ class VariableValidator {
                 self.errors.push(getErrorDetailRecord(message, location));
             }
         } else {
+            __InputValue? inputValue = getInputValueFromArray(inputValues, argumentNode.getName());
+            if inputValue is __InputValue {
+                __Type? inputFieldType = getOfType(inputValue.'type);
+                __InputValue[]? inputFieldValues = inputFieldType?.inputFields;
+                inputValues = inputFieldValues is __InputValue[] ? inputFieldValues : [];
+            }
```

Comment:
What's the `else` logic here? What happens when the input value is `()`?

Reply:
This one is also skipped.